modules/RTC/TraceablePeerConnection.js

```diff
@@ -2008,6 +2034,36 @@ TraceablePeerConnection.prototype.generateNewStreamSSRCInfo = function(track) {
 };
 
 /**
+ * Determines whether the local ICE candidate address is a relay one.
+ *
+ * @param {Array} addressArray an array in the form [ ip address, port ].
+ * @param {String} candidateType the candidate type: One of "host", "srflx",
+ * "prflx", or "relay"
+ * @return {boolean} true if the ICE candidate is a relayed one, otherwise
+ * false.
+ */
+TraceablePeerConnection.prototype.checkLocalCandidateType
+= function(addressArray, candidateType) {
+    return isRelay(
+        this.peerconnection.localDescription, addressArray, candidateType);
+};
+
+/**
+ * Determines whether the remote ICE candidate address is a relay one.
+ *
+ * @param {Array} addressArray an array in the form [ ip address, port ].
+ * @param {String} candidateType the candidate type: One of "host", "srflx",
+ * "prflx", or "relay"
+ * @return {boolean} true if the ICE candidate is a relayed one, otherwise
+ * false.
+ */
+TraceablePeerConnection.prototype.checkRemoteCandidateType
+= function(addressArray, candidateType) {
+    return isRelay(
+        this.peerconnection.remoteDescription, addressArray, candidateType);
```

Comment:
As per our discussion offline, it would be good to add a comment here which will say that we explicitly go around 'remoteDescription' getter ('localDescription' in the other method) in order to go around SDP transformation chain since local addresses are not modified currently anyway. Just to warn others to not change it the other way by mistake.